cloud_functions/climateiq_merge_output_cf/main.py

```diff
@@ -1,7 +1,98 @@
-from cloudevents import http
+import collections
+import csv
+import re
+
+import flask
+from google.cloud import storage
 import functions_framework
 
 
-@functions_framework.cloud_event
-def merge_scenario_predictions(cloud_event: http.CloudEvent) -> None:
-    return
+# Bucket name, where prediction outputs are stored.
+BUCKET_NAME = "climateiq-predictions"
+# File name pattern for the CSVs for each scenario and chunk.
+CHUNK_FILE_NAME_PATTERN = (
+    r"(?P<run_id>\w+)/(?P<prediction_type>\w+)/(?P<model_id>\w+)/"
```

Comment:
This is a comment on your PR but my thinking is that we want to be able to trace the outputs to a specific batch ID in Vertex AI.

Reply:
I believe the model_id represents that based on an earlier conversation with Walt. A given version of a model would only run once per study_area/scenario_chunk. 

@waltaskew to confirm though

Reply:
I think it's a question of if we re-ran the same model, would we want it to overwrite old outputs or produce new ones? Including the batch ID would do the latter.

Reply:
I think keeping the old outputs makes things easier to debug (especially since we haven't tested all this e2e yet). And there will never be a state where half the files in one set are from one run and half of them are from another (e.g., mid re-run). We don't need to retain the old outputs forever - we can set up a TTL and auto-delete the files once they reach a certain [age](https://cloud.google.com/storage/docs/lifecycle#age). The downside, as mentioned above, is that USL would have to ensure they trigger this cloud function before the files reach the TTL.